fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/ValueIndexLikeMatchCandidate.java

```diff
@@ -98,8 +98,7 @@ default List<MatchedOrderingPart> computeMatchedOrderingParts(@Nonnull MatchInfo
 
             builder.add(
                     MatchedOrderingPart.of(value,
-                            comparisonRange == null ? ComparisonRange.Type.EMPTY : comparisonRange.getRangeType(),
-                            queryPredicate,
+                            comparisonRange == null ? ComparisonRange.Type.EMPTY : comparisonRange.getRangeType(), queryPredicate,
```

Comment:
Didn't you say this was a remnant of some previous iteration?

Reply:
Not sure what you mean, I reverted the remnant part, the revert left the code styled differently apparently, I can also revert that unless you mean something else?

Reply:
Nevermind. I saw ghosts.